tasks-by-users/Kiran-html/01-Project-flexbox/assets/css/style.css

```diff
@@ -0,0 +1,11 @@
+.container{
```

Comment:
just a note, you can add as reset. Keep this in mind next time ;) 
```css
* {
  margin: 0;
  padding:0;
  box-sizing: border-box;
}
```

Reply:
Noted.